seccomp/src/lib.rs

```diff
@@ -1280,7 +1289,10 @@ mod tests {
                     code: 21,
                     jt: 1,
                     jf: 0,
+                    #[cfg(target_arch = "x86_64")]
```

Comment:
The following literal hex values seem to be the actual computed values for `AUDIT_ARCH_X86_64` and `AUDIT_ARCH_AARCH64`. Would be nice to quickly mention this in a comment (or use them by name rather than by value), so it's a bit clearer what's going on.

Reply:
Done. Used the name. Take a look.